prime-angular-frontend/src/app/modules/site-registration/pages/overview/overview.component.ts

```diff
@@ -97,4 +110,27 @@ export class OverviewComponent implements OnInit {
       this.showSubmission = !this.site.submittedDate;
     }
   }
+
+  /**
+   * @description
+   * Infer whether user was required to accept an Organization Agreement.
+   * User saw an Organization Agreement if the current Site has a care setting that is unique
+   * for that Organization.
+   */
+  private determineOrgAgreementRequired$(): Observable<SiteListViewModel[]> {
+    const currentCareSettingCode: number = this.site.careSettingCode;
+
+    return this.siteResource.getSites(this.organization.id).pipe(
+      tap(sites => {
```

Comment:
I don't think you will want to use `tap`.  Tap can be used for logging or performing side effects like you are currently, but it looks like you just want to get a value passed through the observable to the subscribe method.  If I wanted to do this I would use `map` in the pipe and within the map I would `site.reduce` to a number that you return from the map, and it will be passed to the subscribe.